lib/money/money.rb

```diff
@@ -191,6 +191,8 @@ def to_s(style = nil)
     case style
     when :legacy_dollars
       sprintf("%.2f", value)
+    when :with_currency
+      "#{value.to_s('F')} #{currency.to_s}"
```

Comment:
Is there a use case for this now? It's fairly easy for someone to do this on their own and most likely this needs to be locale specific. In any case, if there is a use case you'd need
```
"#{to_s(:amount)} #{currency.to_s}"
```
to have the correct number of decimals